include/zephyr/posix/signal.h

```diff
@@ -47,13 +47,15 @@ extern "C" {
 #define SIGSYS    31 /**< Bad system call */
 
 #define SIGRTMIN 32
-#define SIGRTMAX (SIGRTMIN + RTSIG_MAX)
-#define _NSIG (SIGRTMAX + 1)
-
-BUILD_ASSERT(RTSIG_MAX >= 0);
+#if defined(CONFIG_POSIX_REALTIME_SIGNALS) || defined(__DOXYGEN__)
```

Comment:
Why do we need doxygen check here as there is no documentation changes involved?

Reply:
No need to remove it, I was just curious.